Styles/Style.css

```
@@ -0,0 +1,420 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300;400;700;900&display=swap');
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        4.3 Main
+        4.4 Notification
+        4.5 Footer
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+header{
+    font-family: 'Merriweather', serif;
+}
+
+body {
+    font-family: 'Poppins', sans-serif;
+}
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-image: linear-gradient(to right, #4845E4, #5722CD);
+    color: #fff;
+    padding: 1rem 1rem 10rem 1rem;
+    text-align: center;
+    position: relative;
+}
+
+header:after{
+    content: '';
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    background: inherit;
+    z-index: -1;
+    bottom: 0;
+    transform-origin: left bottom;
+    transform: skewY(2deg);
+}
+
+
+header nav {
+    display: flex;
+    align-self: flex-end;
+    margin-right: 10rem;
+}
+
+nav ul {
+    display: flex;
+    gap: 2rem;
+}
+
+ul a {
+    color: #ffffff;
+    text-decoration: none;
+    font-size: 0.8rem;
+}
+
+ul li{
+    list-style-type: none;
+}
+
+.content{
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: center;
+    flex: 1 1 15rem;
+    flex-wrap: wrap;
+    width: 30%;
```

Deze had wel wat groter gemogen wat mij betreft
```suggestion
    width: 70%;
```

---

ik snap wat je bedoeld, toen ik dit maakte had ik hem echter openstaan op mijn grote scherm en dan is 30% prima op mijn kleinere scherm is het inderdaad aan de kleine kant. Oftewel meerdere schermen testen :)